core/crypto/src/key_conversion.rs

```diff
@@ -24,11 +24,11 @@ pub fn convert_public_key(key: &signature::ED25519PublicKey) -> Option<vrf::Publ
 }
 
 pub fn convert_secret_key(key: &signature::ED25519SecretKey) -> vrf::SecretKey {
-    let b = ed25519_dalek::ExpandedSecretKey::from(
-        &ed25519_dalek::SecretKey::from_bytes(&key.0[..32]).unwrap(),
+    let b = ed25519_dalek::hazmat::ExpandedSecretKey::from(
```

Comment:
Especially this section. I couldn’t find another function in ed25519-dalek that seemed to do the same hashing+resetting bits logic as ExpandedSecretKey::from; but maybe I missed something?

All in all, I’d feel much better if we didn’t have to turn on the hazmat feature of ed25519-dalek.